### PR TITLE
⭐️ allow users to ignore .terraform directory

### PR DESCRIPTION
**Pull request by chris-rock**
fixes  #605

### CODE REVIEW

File: providers/terraform/connection/hcl_manifest.go [hunk 1]
```diff
@@ -51,6 +51,15 @@ func NewHclConnection(id uint32, asset *inventory.Asset) (*Connection, error) {
 
 func newHclConnection(path string, asset *inventory.Asset) (*Connection, error) {
 	// NOTE: right now we are only supporting to load either state, plan or hcl files but not at the same time
+	if len(asset.Connections) != 1 {
+		return nil, errors.New("only one connection is supported")
+	}
+
+	confOptions := asset.Connections[0].Options
+	includeTerraform := true
```
Comment (czunker) [inline providers/terraform/connection/hcl_manifest.go]: nit: Should we rename this to `includeDotTerraform`? As it is now, it somehow reads strange to not include terraform in a terraform connection.

**Review by czunker — APPROVED** [review]
Thanks @chris-rock Gave this a try with some other terraform manifest and it works as expected.

File: providers/terraform/connection/hcl_manifest.go (hunk 1 above)
Comment (chris-rock) [reply providers/terraform/connection/hcl_manifest.go]: Great idea

**Review by czunker — APPROVED** [review]
Thanks @chris-rock

**Pull request merged**